Sources/SwiftInspectorCommands/Tests/TypealiasCommandSpec.swift

```diff
@@ -50,24 +50,44 @@ final class TypealiasCommandSpec: QuickSpec {
       }
 
       context("with no --name argument") {
-        it("fails") {
-          let result = try? TestTask.run(withArguments: ["typealias", "--path", path])
-          expect(result?.didFail) == true
+        var result: TaskStatus?
+        beforeEach {
+          result = try? TestTask.run(withArguments: ["typealias", "--path", path])
+        }
+
+        it("succeeds") {
+          expect(result?.didSucceed) == true
+        }
+
+        it("outputs the typealias information") {
+          expect(result?.outputMessage).to(contain("SomeAlias"))
+        }
+
+        it("does not output the typelias identifiers") {
+          expect(result?.outputMessage).toNot(contain("SomeType"))
         }
       }
 
       context("with an empty --name argument") {
-        it("fails") {
+        it("succeeds") {
```

Comment:
Do we really want to support this? Passing an empty name feels like misuse? But I guess maybe the argument for doing this would be that it's empty if you don't specify it?

Reply:
Ah I see below.